backend/sozisel/lib/sozisel/model/quizzes/quiz_question.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Sozisel.Model.Quizzes.QuizQuestion do
+  use Sozisel.Model.Schema
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          question: String.t(),
+          answers: [String],
```

Comment:
Dont forget to add `.t()` to types as `String` in this case is just a plain atom, not a type spec.